react/frontpage/components/ShowBlurb.scss

```diff
@@ -47,14 +61,41 @@ Sass styling for show blurb on shows page (with grid)
 .blurbText {
   font-size: 12px;
   font-weight: lighter;
+  margin-top: 10px;
 }
 
 .dot {
-  background-color: grey;
+  background-color: rgba(255,0,0,0.45);
   width: 10px;
   height: 10px;
   border-radius: 50%;
   display: inline-block;
   margin-left: 5px;
+  margin-right: 10px;
   position: relative;
+  left: 1;
+}
+
+.fa {
+  margin-right: 7pt;
+}
+
+.facebookLogo:hover {
+  color: $facebook !important;
+  opacity: 0.95;
+}
+
+.tumblrLogo:hover {
+  color: $tumblr !important;
+  opacity: 0.95;
+}
+
+.soundcloudLogo:hover {
+  color: $soundcloud !important;
+  opacity: 0.95;
+}
+
+.mixcloudLogo:hover {
+  color: $mixcloud !important;
+  opacity: 0.95;
 }
```

Comment:
Newlines [are cool](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).